cmd/ncproxy/run.go

```diff
@@ -29,12 +31,21 @@ type nodeNetSvcConn struct {
 	grpcConn *grpc.ClientConn
 }
 
+type networkProxyStore struct {
+	networkStore  *ncproxy.NetworkStore
+	endpointStore *ncproxy.EndpointStore
+}
```

Comment:
This shouldn't be needed as discussed. We should just be able to reach out to ncproxy for any endpoint and network details.